android-file-chooser/src/main/java/ir/sohreco/androidfilechooser/ExternalStorageNotAvailableException.java

```diff
@@ -1,7 +1,7 @@
 package ir.sohreco.androidfilechooser;
 
-public class ExternalStorageNotAvailableException extends Exception {
-    public ExternalStorageNotAvailableException() {
+class ExternalStorageNotAvailableException extends RuntimeException {
```

Comment:
I think that the idea of checked exception i n java is not an ideal. If user of this class not sure that external storage is available, then he must check it before using file picker dialog. But it just my opinion, if you sure that this exception must be checked, we can discuss it.